src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_StorageFile.cs

```diff
@@ -323,6 +323,28 @@ public async Task When_CreateFile_GenerateUnique_Exists_Folder()
 			}
 		}
 
+		[TestMethod]
+		public async Task When_Open_By_Encoded_URI_With_Space()
+		{
+			var uri = new Uri($"ms-appx:///With Space.txt");
```

Comment:
Can you make a runtime test which specifically calls `GetFileFromApplicationUri`? You can add such file to the `Assets` of the Samples app shared project to test it

Reply:
Oops! My mistake.